### ISSUE TITLE
Somebody Copied Codes, Exported it, Now Trying to Sell with Different Name on Google Play

### ISSUE THREAD

**Issue by sifsty** [LibreShift/red-moon#214]
[GhostScreen -> https://play.google.com/store/apps/details?id=net.lgdestek.ghostscreenfix](https://play.google.com/store/apps/details?id=net.lgdestek.ghostscreenfix)  on Google Play.

![image 2018-01-08 10 20 35 pm](https://user-images.githubusercontent.com/24463622/34688134-21289684-f4c2-11e7-9b50-f8585f89e3e0.jpg)

I can easily say the developer (actually we can't say developer) copy your all codes, exported it then he/she is trying to sell it on Google Play Store. This app and should remove from Google Play.

**Comment by smichel17** [comment]
Indeed. If they are using our code, they need to provide the source and attribution to this project. I'll do the same thing as in #161 and maybe @raatmarien would like to, as well?

**Comment by sifsty** [comment]
Update: It is still in the Google Play Store.

**Comment by smichel17** [comment]
Yeah, I forgot to do this. I'll do it now

**Comment by smichel17** [comment]
Done.

**Comment by sifsty** [comment]
## Sorry but you need to reopen the issue. The app moved instead of to be removed.
Under swiow username, even you see same link, same name and same screenshots.
https://play.google.com/store/apps/details?id=net.lgdestek.ghostscreenfix

It would be frustrating, someone trying to money from you open source app. Is there a thing which we can do?

![image 2018-02-05 8 04 46 pm](https://user-images.githubusercontent.com/24463622/35817947-d14b62c8-0aaf-11e8-9b25-6fba5175a958.jpg)

**Comment by smichel17** [comment]
I filed another dmca takedown request, so it should be down shortly.

> It would be frustrating, someone trying to money from you open source app.

I would like to clarify one thing: I have to problem with people making money off FLO software. In fact, [I would like more of that](https://wiki.snowdrift.coop). Not to mention, Red Moon itself is $2 on Google Play (and free on F-Droid).

If someone wanted to sell a Red Moon clone *and they released the source code, according to the GPL,* then I would not do anything about it. If they added features, I might encorporate them back into Red Moon. However, breaking the license and adding ads is not an acceptable way of doing that.

>  Is there a thing which we can do?

The only thing to do is sending dmca takedown notices each time a new one pops up. The upside is that when an app is taken down, Google will charge them back for any profits they made and after the second time they will have to make a new account, which costs $25.

I presume they have some way to avoid the chargeback and they make more profits than the $25. However, I also know it takes them more effort to set up a new account than it takes me to report the apps, so if we find and report them quickly, before too many people purchase them, we can hopefully make it not worth their time and they will find some other app to rip off.

**Comment by Poussinou** [comment]
@LibreShift

I found another at https://play.google.com/store/apps/details?id=nightmode.ardentic.screenfilter and yet another one at https://play.google.com/store/apps/details?id=com.zerocode.redmoon.lite

Maybe you could fill the DMCA takedown request for this ones as well 👍

**Comment by Poussinou** [comment]
@LibreShift There is another one here: https://play.google.com/store/apps/details?id=net.sunnite.blue

**Comment by Poussinou** [comment]
Yet another one here: https://play.google.com/store/apps/details?id=com.creatoro.nightscreenfilter

**Comment by smichel17** [comment]
I need to make a template for myself so I don't need to write these out manually each time.

**Comment by smichel17** [comment]
If anyone wants to write out such a template, that would be nice.
